### PR TITLE
Adding express

### PR DESCRIPTION
**Pull request by brogers32**
add frontend to backend

### CODE REVIEW

**Review by Svedek — CHANGES_REQUESTED** [review]
Fix syntax

File: server/database/database.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+async function getLoginInfo(email) {
+    const query = "SELECT * FROM login_info WHERE email = ?";
+    const { rows } = await pool.query(query, [email]);
+    return rows;
+}
+
+async function getUser(username) {
+    const query = "SELECT * FROM user WHERE username = ?";
+    const { rows } = await pool.query(query, [username]);
```
Comment (Svedek) [inline server/database/database.js]: syntax here is wrong

Comment (brogers32) [reply server/database/database.js]: are you referring to the curly braces where I should have used brackets?

**Review by brogers32 — COMMENTED** [review]
Looks good!

**Pull request merged**